### ISSUE TITLE
line 6 ImportError: No Module Named 'drozer' 

### ISSUE THREAD

**Issue by schwann007** [mwrlabs/drozer#174]
Recently I am working on a project focus on Android malware detection. To decompiIe the malware, I plan to install Adroguard. However,  when I try to install drozer, system returns error. I try to figure it out, but it still doesn't work.
Installation Environment:
Windows 10;
Python 3;
Android SDK
Adb
JDK
![3](https://cloud.githubusercontent.com/assets/13725060/10623353/4aaa99c0-775c-11e5-9fe0-fa70de21e3de.jpg)
![4](https://cloud.githubusercontent.com/assets/13725060/10623360/4ff1f662-775c-11e5-9941-61f80865c8aa.jpg)

**Comment by HenryHoggard** [comment]
@JosephRedfern Any ideas?

**Comment by metall0id** [comment]
Hi,

drozer uses Python 2.7 and not Python 3

I often get error messages like this when things have been installed in the site-packages folder for Python 2.7 and I am trying to use it from Python 3

TL;DR: Please make sure you have Python 2.7 installed

**Comment by HenryHoggard** [comment]
@metall0id Still get this with python 2.7.

**Comment by HenryHoggard** [comment]
You have to set your PYTHONPATH (https://github.com/mwrlabs/drozer/wiki/Running-on-the-edge).
